docs/architecture/adr-003-dynamic-capability-store.md

```diff
@@ -85,23 +85,23 @@ It MUST be called before `InitialiseAndSeal`.
 
 ```go
 func (ck CapabilityKeeper) ScopeToModule(moduleName string) ScopedCapabilityKeeper {
-	if k.sealed {
-		panic("cannot scope to module via a sealed capability keeper")
-	}
-
-	if _, ok := k.scopedModules[moduleName]; ok {
-		panic(fmt.Sprintf("cannot create multiple scoped keepers for the same module name: %s", moduleName))
-	}
-
-	k.scopedModules[moduleName] = struct{}{}
-
-	return ScopedKeeper{
-		cdc:      k.cdc,
-		storeKey: k.storeKey,
-		memKey:   k.memKey,
-		capMap:   k.capMap,
-		module:   moduleName,
-	}
+ if k.sealed {
```

Comment:
I, personally, don't like that indentation. Have you run `make build-docs` to see how it renders in our docs?